### PR TITLE
Line editing for test.rst

### PR DESCRIPTION
**Pull request by christinerose**

### CODE REVIEW

File: doc/tests.rst [hunk 1]
```diff
@@ -48,36 +49,38 @@ exec`` to run the test executable (for the sake of this example,
 
 .. code:: bash
 
-   dune exec project/tests/myTest.exe
+dune exec project/tests/myTest.exe
```
Comment (rgrinberg) [inline doc/tests.rst:52]: The indention here is intentional. It's needed for sphinx to display this line as code.

Comment (christinerose) [reply doc/tests.rst:52]: Thank you for catching that!

Comment (christinerose) [reply doc/tests.rst]: Okay! I think I fixed it!

**Comment by christinerose** [comment]
You're so welcome. It's my pleasure to be of help!

IRT the contractions, I find the we'll and don't more natural than 'we will' and 'do not'. I generally reserve 'we will' when I want to emphasize that "we will indeed!!" do something. I also find that when reading, 'don't' is seen as the negative whereas sometimes the 'not' is overlooked in 'do not'. Same with 'will not,' so I prefer won't. 

All that said, it really is a personal preference. Contractions feel more natural in most circumstances to me.

**Comment by christinerose** [comment]
It looks like I signed off on all the commits, so I don't know why I'm not able to merge. Would you please have a quick look @jeremiedimino or @rgrinberg? Please also let me know if there are any other comments or if it's okay to squash/merge.

**Comment by jeremiedimino** [comment]
@christinerose if you click on the `Details` link in front of the DCO item in the list of checks, you can see which commits are missing a sign off. I've manually clicked on "Set DCO to pass", so you should be able to squash and merge your PR.

**Review by jeremiedimino — APPROVED** [review]
Looks good!

**Pull request merged**